spring-cloud-starter-tencent-polaris-discovery/src/test/java/com/tencent/cloud/polaris/discovery/PolarisDiscoveryClientTest.java

```diff
@@ -41,8 +40,7 @@
  *
  * @author Haotian Zhang
  */
-@RunWith(PowerMockRunner.class)
-@PowerMockIgnore("javax.management.*")
+@RunWith(SpringRunner.class)
```

Comment:
> Why do you change the RunWith CLASS?

Used @RunWith(MockitoJUnitRunner.class) instead of @RunWith(PowerMockRunner.class) ,
After spring boot 2.6.x , `spring-boot-starter-test` depend with `mockito-core: 4.0.0`, 
and `powermock` framework is not compatible with `mockito-core: 4.0.0` .

So ,can i removed `powermock` test framework ?

Reply:
Remove powermock.